src/panels/config/integrations/integration-panels/ozw/ozw-network-nodes.ts

```diff
@@ -0,0 +1,176 @@
+import "@material/mwc-fab";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import memoizeOne from "memoize-one";
+import { mdiCheck, mdiAlert, mdiServerNetwork, mdiNetwork } from "@mdi/js";
+import { navigate } from "../../../../../common/navigate";
+import { DataTableColumnContainer } from "../../../../../components/data-table/ha-data-table";
+import "../../../../../components/ha-card";
+import "../../../../../components/ha-icon-next";
+import "../../../../../components/buttons/ha-call-service-button";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant, Route } from "../../../../../types";
+import "../../../ha-config-section";
+import type { PageNavigation } from "../../../../../layouts/hass-tabs-subpage";
+import "../../../../../layouts/hass-tabs-subpage";
+import "../../../../../layouts/hass-tabs-subpage-data-table";
+import "@material/mwc-button/mwc-button";
+import { OZWDevice, fetchOZWNodes } from "../../../../../data/ozw";
+import { computeTail } from "./ozw-config-router";
+
+export interface NodeRowData extends OZWDevice {
+  node?: NodeRowData;
+  id?: number;
+}
+
+@customElement("ozw-network-nodes")
+class OZWNetworkNodes extends LitElement {
+  @property({ type: Object }) public hass!: HomeAssistant;
+
+  @property({ type: Object }) public route!: Route;
+
+  @property({ type: Boolean }) public narrow!: boolean;
+
+  @property({ type: Boolean }) public isWide!: boolean;
+
+  @property() public configEntryId?: string;
+
+  @property() public ozw_instance = 0;
+
+  @internalProperty() private _nodes: OZWDevice[] = [];
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    if (this.ozw_instance <= 0) {
+      navigate(this, "/config/ozw/dashboard", true);
+    } else if (this.hass) {
+      this._fetchData();
+    }
+  }
+
+  private async _fetchData() {
+    this._nodes = await fetchOZWNodes(this.hass!, this.ozw_instance);
+  }
+
+  private _formattedNodes = memoizeOne((nodes: OZWDevice[]) => {
+    let outputNodes: NodeRowData[] = nodes;
+
+    outputNodes = outputNodes.map((node) => {
+      return {
+        ...node,
+        id: node.node_id,
+      };
+    });
+
+    return outputNodes;
+  });
+
+  private _columns = memoizeOne(
+    (narrow: boolean): DataTableColumnContainer =>
+      narrow
```

Comment:
if the hidden columns PR is merged before this, we should instead of not sending the column into the object, hide them with `hidden` property